### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DavidSouther**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,27 +14,47 @@ Ailly's best feature is rapid prompt engineering iteration. By keeping your prom
...
-   - Include system prompts, level setting expectations. etc.
-3. Create more numbered files, such as `content/01_big_point.md` and `content/02_second_point.md`.
-4. Run Ailly using NodeJS: `npx @ailly/cli --root content`
+1. Create a folder named `jokes`.
+2. Create a file named `10_chickens.md` with "Tell me a joke about chickens" as the content.
+3. Run Ailly using NodeJS: `npx @ailly/cli --root content`
+   - See the joke in `10_chickens.md.ailly.md`
+2. Create a file named `content/.aillyrc` with "You are a farmer writing jokes for your other barnyard animals."
+   - Include other system prompts, level setting expectations. etc.
+   - Run Ailly with the same command, and see how thr joke changes.
```
Comment (cpyle0819) [inline README.md:23]:
```suggestion
   - Run Ailly with the same command, and see how the joke changes.
```

File: README.md [hunk 2]
```diff
@@ -14,27 +14,47 @@ Ailly's best feature is rapid prompt engineering iteration. By keeping your prom
...
-1. Create a folder named `content`.
-2. Create a file named `content/.aillyrc`, and put your top-level prompt instructions in the file.
-   - Include system prompts, level setting expectations. etc.
-3. Create more numbered files, such as `content/01_big_point.md` and `content/02_second_point.md`.
-4. Run Ailly using NodeJS: `npx @ailly/cli --root content`
+1. Create a folder named `jokes`.
+2. Create a file named `10_chickens.md` with "Tell me a joke about chickens" as the content.
+3. Run Ailly using NodeJS: `npx @ailly/cli --root content`
+   - See the joke in `10_chickens.md.ailly.md`
+2. Create a file named `content/.aillyrc` with "You are a farmer writing jokes for your other barnyard animals."
```
Comment (cpyle0819) [inline README.md:21]: Any greymatter here or just the plaintext?

File: README.md [hunk 3]
```diff
@@ -14,27 +14,47 @@ Ailly's best feature is rapid prompt engineering iteration. By keeping your prom
...
+3. Create more numbered files, such as `content/20_knock_knock.md` with "Turn the chicken joke into a knock knock joke."
+4. Run Ailly using NodeJS: `npx @ailly/cli --root content content/20_knock_knock.md`
+  - Ailly will send each file to the configured LLM engine and write its result.
+  - `20_knock_knock.md.ailly.md` will have the new knock knock joke based on thr chickenjome it first wrote!
+
+### System Context
+
+System prompts provide grounding and background information for LLMs.
+There are a number of techniques and "best practices" for developing LLM system prompts.
+In Ailly, these are in files with the name `.aillyrc`, and apply top all files in the current folder when using Ailly.
```
Comment (cpyle0819) [inline README.md:33]:
```suggestion
In Ailly, these are in files with the name `.aillyrc`, and apply to all files in the current folder when using Ailly.
```

File: README.md [hunk 4]
```diff
@@ -14,27 +14,47 @@ Ailly's best feature is rapid prompt engineering iteration. By keeping your prom
...
 - **`isolated`**: `boolean` If `true`, the LLM inference only includes the system prompt, and not the prior context in this folder.
 - **`context`** `conversation` | `folder` | `none`
-  - `conversation` (default) loads files from the root folder and includes them alphabetically, chatbot history style, before the current file.
-  - `folder` includes all files in the folder at the same level as the current file.
+  - `conversation` (default, unless editing) loads files from the root folder and includes them alphabetically, chatbot history style, before the current file.
+  - `folder` includes all files in the folder at the same level as the current file. (Default when editing.)
   - `none` includes no additional content (including no system context).
+- **`parent`** `root` | `always` | `never`
+  - `root` (default) start the chain of system prompts from here.
+  - `always` include the .aillyrc file in the parent directory as part of this system prompt.
```
Comment (cpyle0819) [inline README.md:54]: Is this the immediate parent only? Or are grandparents considered?

**Review by cpyle0819 — CHANGES_REQUESTED** [review]
A few typos and some consistency concerns.

File: README.md (hunk 2 above)
Comment (DavidSouther) [reply README.md:21]: Just plaintext at this point in the Quickstart.

File: README.md (hunk 4 above)
Comment (DavidSouther) [reply README.md:54]: Immediate parent, which then goes through the same process of default root, can be set to none or parent.

File: README.md [hunk 5]
```diff
@@ -14,27 +14,47 @@ Ailly's best feature is rapid prompt engineering iteration. By keeping your prom
 
 To get started, follow these steps:
 
-1. Create a folder named `content`.
-2. Create a file named `content/.aillyrc`, and put your top-level prompt instructions in the file.
-   - Include system prompts, level setting expectations. etc.
-3. Create more numbered files, such as `content/01_big_point.md` and `content/02_second_point.md`.
-4. Run Ailly using NodeJS: `npx @ailly/cli --root content`
+1. Create a folder named `jokes`.
+2. Create a file named `10_chickens.md` with "Tell me a joke about chickens" as the content.
```
[the remark this answers is not in the document]
Comment (DavidSouther) [reply README.md:18]: Yep it should be --root jokes, thank you.

**Review by cpyle0819 — APPROVED** [review]

**Pull request merged**